solo-chains/runtime/starlight/src/lib.rs

```diff
@@ -1430,6 +1432,39 @@ impl pallet_data_preservers::Config for Runtime {
     type MaxParaIdsVecLen = MaxLengthParaIds;
 }
 
+parameter_types! {
+    pub StarlightBondAccount: AccountId32 = PalletId(*b"StarBond").into_account_truncating();
+    pub PendingRewardsAccount: AccountId32 = PalletId(*b"PENDREWD").into_account_truncating();
+    // The equation to solve is:
+    // initial_supply * (1.05) = initial_supply * (1+x)^5_259_600
+    // we should solve for x = (1.05)^(1/5_259_600) -1 -> 0.000000009 per block or 9/1_000_000_000
+    // 1% in the case of dev mode
+    // TODO: check if we can put the prod inflation for tests too
+    // TODO: better calculus for going from annual to block inflation (if it can be done)
+    pub const InflationRate: Perbill = runtime_common::prod_or_fast!(Perbill::from_parts(9), Perbill::from_percent(1));
```

Comment:
This inflation rate will change in future correct? If so, can you add a todo?

Reply:
Added in 8695f800e855732efd38ed4b48968fde53f08728.

Reply:
_fixed_